rsts/community/contribute.rst

```diff
@@ -156,36 +267,25 @@ Component Reference
 
 .. list-table::
 
-    * - `Repo <https://github.com/lyft/flytesnacks>`__
+    * - `Repo <https://github.com/flyteorg/flytesnacks>`__
     * - **Purpose**: Examples, Tips, and Tricks to use Flytekit SDKs
-    * - **Language**: Python (In future, Java shall be added)
-    * - **Setup**:
-
-        * If the Python code has to be tested, run it locally
-        * If the Python code has to be tested in a cluster:
-            * Run the ``make start`` command in the root directory of the flytesnacks repo
-            * Visit https://localhost:30081 to view the Flyte console consisting of the examples present in ``flytesnacks/cookbook/core`` directory
-            * To fetch the new dependencies and rebuild the image, run ``make register``
+    * - **Language**: Python (In the future, Java examples will be added)
+    * - **Guidelines**: Refer to the `Flytesnacks Contribution Guide <https://docs.flyte.org/projects/cookbook/en/latest/contribute.html>`__
 
 ``flytectl``
 ************
 
 .. list-table::
 
-    * - `Repo <https://github.com/lyft/flytectl>`__
-    * - **Purpose**: A Standalone Flyte CLI
+    * - `Repo <https://github.com/flyteorg/flytectl>`__
+    * - **Purpose**: A standalone Flyte CLI
     * - **Language**: Go
-    * - **Setup**:
+    * - **Guidelines**: Refer to the `FlyteCTL Contribution Guide <https://docs.flyte.org/projects/flytectl/en/stable/contribute.html>`__    
 
-        * Check for the Makefile in the root repo
-        * Run the following commands:
-            * ``make generate``
-            * ``make test_unit``
-            * ``make link``    
+🐞 File an Issue
+================
 
-Issues
-======
-`GitHub Issues <https://github.com/flyteorg/flyte/issues>`__ is used for issue tracking. There are a variety of issue types available that you could use while filing an issue.
+`GitHub Issues <https://github.com/flyteorg/flyte/issues>`__ is used for issue tracking. Following are the issue types available that you could use while filing an issue.
 
 * `Plugin Request <https://github.com/flyteorg/flyte/issues/new?assignees=&labels=untriaged%2Cplugins&template=backend-plugin-request.md&title=%5BPlugin%5D>`__
 * `Bug Report <https://github.com/flyteorg/flyte/issues/new?assignees=&labels=bug%2C+untriaged&template=bug_report.md&title=%5BBUG%5D+>`__
```

Comment:
Can we also ask the reporter to add relevant labels which include the component name which they think should triage this first.

Reply:
Added this, thanks!